group-generators/helpers/data-providers/alchemy/index.ts

```diff
@@ -83,48 +84,47 @@ export class AlchemyProvider {
   }
 
   /**
-   * Retrieves the owners of all NFTs in a collection that match a tokenId
+   * Retrieves the owners of all NFTs in a collection that match the tokenids
    * @param {Object} params - The parameters for fetching owners by token IDs.
    * @param {string} contractAddress - The address of the NFT contract.
    * @param {string} chain - The name of the blockchain network.
-   * @param {string} TokenId - The tokenId of the NFT to search for.
+   * @param {string[]} TokenIds -  An array token IDs to fetch owners for.
    * @returns {Promise<FetchedData>} - A Promise that resolves to an object with the owner addresses as keys and a value of 1.
    */
-  public async getOwnersForOneTokenId({
```

Comment:
Make sure that on groups are using this function:

I saw that: `example-layer3` used it

For all the others it seems they used only `getOwnersForCollection` so it's okay :)

Reply:
Great! That's my example, I'll update both.